yellowbrick/features/base.py

```diff
@@ -124,6 +130,12 @@ def fit(self, X, y=None, **fit_params):
 ## Data Visualizers
 ##########################################################################
 
+class Target_Type(Enum):
```

Comment:
Minor typo, `Target_Type` should be `TargetType`, variable names are:

- all caps snake case for constants
- camel case for classes 
- all lower snake case for functions, methods, and other variables